### ISSUE TITLE
Replace jQuery SrollDown with a scroll-to-anchor

### ISSUE THREAD

**Issue by nearestnabors** [rachelnabors/alice-in-videoland#7]

**Comment by nearestnabors** [comment]
Use [`scroll-behavior`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior): https://gomakethings.com/how-to-animate-scrolling-to-anchor-links-with-one-line-of-css/
```css
/**
 * Disable smooth scrolling when users have prefers-reduced-motion enabled
 */
@media screen and (prefers-reduced-motion: reduce) {
	html {
		scroll-behavior: auto;
	}
}
```

**Comment by nearestnabors** [comment]
Closed via 035afd6
